src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/build/Microsoft.DotNet.UpgradeAssistant.Abstractions.props

```diff
@@ -1,5 +1 @@
-﻿<Project>
-  <PropertyGroup>
-    <ZipExtensionMethod>false</ZipExtensionMethod>
-  </PropertyGroup>
-</Project>
```

Comment:
Do you need this file? seems like the reference to the props file has been removed from everywhere

Reply:
Yeah - it's for the NuGet package. I moved the properties into the props so we can just reference it once